src/java/org/apache/cassandra/exceptions/RequestFailureReason.java

```diff
@@ -71,10 +72,11 @@ public enum RequestFailureReason
 
     private static final Map<Integer, RequestFailureReason> codeToReasonMap = new HashMap<>();
     private static final Map<Class<? extends Throwable>, RequestFailureReason> exceptionToReasonMap = new HashMap<>();
-    private static final int REASONS_WITHOUT_EXCEPTIONS = 3; // UNKNOWN, NODE_DOWN, and READ_TOO_MANY_INDEXES
 
     static
     {
+        EnumSet<RequestFailureReason> withoutExceptions = EnumSet.of(UNKNOWN, NODE_DOWN, READ_TOO_MANY_INDEXES);
```

Comment:
it was very annoying to get the root cause from the error, so switched to set diff to tell you what values are the problem